src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/layout/formLayoutReducer.ts

```diff
@@ -15,6 +15,7 @@ export interface ILayoutState {
   layouts: ILayouts;
   error: Error;
   uiConfig: IUiConfig;
+  layoutset: ILayoutSets;
```

Comment:
Would `layoutSets` be a better name? The type `ILayoutSets` at least suggests that there might be multiple layoutSets stored here.